package.json

```diff
@@ -8,34 +8,44 @@
 		"yarn": ">=3.1.0"
 	},
 	"scripts": {
-		"dev": "nuxi dev",
-		"build": "nuxi build",
+		"dev": "vue-cli-service serve",
+		"build": "vue-cli-service build",
+		"lint": "vue-cli-service lint",
 		"start": "node .output/server/index.mjs",
 		"lint:js": "eslint --ext \".js,.ts,.vue\" --ignore-path .gitignore .",
 		"lint:prettier": "prettier --check .",
-		"lint": "yarn lint:js && yarn lint:prettier",
 		"lint:fix": "prettier --write --list-different . && yarn lint:js --fix",
-		"typecheck": "nuxi typecheck",
 		"test": "jest"
 	},
-	"devDependencies": {
-		"@nuxtjs/eslint-config-typescript": "^8.0.0",
-		"eslint": "^8.10.0",
-		"eslint-config-prettier": "^8.4.0",
-		"husky": "^7.0.4",
-		"jest": "^27.5.1",
-		"lint-staged": "^12.3.4",
-		"nuxi": "^0.10.1",
-		"nuxt3": "latest",
-		"prettier": "^2.5.1",
-		"typescript": "^4.6.2",
-		"vue-jest": "^3.0.7"
+	"dependencies": {
+		"@algo-builder/algob": "^3.2.0",
+		"@algo-builder/web": "^3.2.0",
+		"@ant-design/icons-vue": "^6.0.1",
+		"algosdk": "^1.10.0",
+		"ant-design-vue": "2.2.1",
```

Comment:
Why downgrading? In the task it was noted to use 3.0.0-beta, which uses Vue.js v3
ant-design-vue 2.2 uses Vue v2

Please make sure we use the latest one: https://github.com/vueComponent/ant-design-vue/releases